### PR TITLE
Update Person.php

### PR DESCRIPTION
**Pull request by RicLeP**
Added schema.org properties

### CODE REVIEW

File: src/ContextTypes/Person.php [hunk 1]
```diff
@@ -10,6 +10,11 @@ class Person extends AbstractContext
      * @var array
      */
     protected $structure = [
-        'name' => null,
```
Comment (boly38) [inline src/ContextTypes/Person.php:13]: tests are not happy as 'name' should exist according to "Properties from Thing" (cf. http://schema.org/Person)

File: src/ContextTypes/Person.php [hunk 2]
```diff
@@ -10,6 +10,11 @@ class Person extends AbstractContext
...
+        'givenName' => null,
+        'familyName' => null,
+        'email' => null,
+        'image' => null,
```
Comment (boly38) [inline src/ContextTypes/Person.php:16]: maybe 'ImageObject::class' ?

**Review by boly38 — CHANGES_REQUESTED** [review]

File: src/ContextTypes/Person.php (hunk 2 above)
Comment (Torann) [reply src/ContextTypes/Person.php:16]: Per the JSON LD spec this is a string or an object. I guess it would be fine as a ImageObject class

**Comment by boly38** [comment]
pull/ #42 should supersede this PR. I made Person to extend Things and add related Unit Tests
BR

---

In my first version, I had missed 'telephone', so I just update a fresh version fo PR 42 with many additionnal Person attributes. Would be great if @Torann could take a look to unlock 2 end-users (maybe more..) ;) :)

**Comment by Torann** [comment]
Per @boly38 PR #42 fixes this 👍

**Pull request closed without merging**